addons/mail/static/src/components/attachment/attachment.scss

```diff
@@ -43,10 +42,25 @@
     justify-content: center;
 }
 
-.o_Attachment_asideItemUnlink.o-pretty {
+.o-pretty {
```

Comment:
`.o-pretty` is top level style? That seems strange and/or an unclear class name.